roles/octavia/tasks/generate_resources.yml

```diff
@@ -152,3 +152,89 @@
     cloud: atmosphere
     image: "{{ octavia_amphora_image_name }}"
   register: _octavia_amphora_image
+
+- name: Deploy Amphora jumpbox
+  run_once: true
+  block:
+    - name: Create amphora ssh key
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          - apiVersion: secretgen.k14s.io/v1alpha1
+            kind: SSHKey
+            metadata:
+              name: amphora-ssh-key
```

Comment:
Let's make this `octavia-amphora-ssh-key` in order to make it clear that it's part of the Octavia things.